docs/source/channel_update_tutorial.rst

```diff
@@ -164,63 +162,26 @@ Org3 peer. From the ``addOrg3`` directory, issue the following command:
+If the command is successful, you will see the creation of the Org3 peer:
 
 .. code:: bash
 
   Creating peer0.org3.example.com ... done
-  Creating Org3cli                ... done
 
 This Docker Compose file has been configured to bridge across our initial network,
-so that the Org3 peer and Org3CLI resolve with the existing peers and ordering
-node of the test network. We will use the Org3CLI container to communicate with
-the network and issue the peer commands that will add Org3 to the channel.
-
-
-Prepare the CLI Environment
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The update process makes use of the configuration translator tool -- configtxlator.
-This tool provides a stateless REST API independent of the SDK. Additionally it
-provides a CLI tool that can be used to simplify configuration tasks in Fabric
-networks. The tool allows for the easy conversion between different equivalent
-data representations/formats (in this case, between protobufs and JSON).
-Additionally, the tool can compute a configuration update transaction based on
-the differences between two channel configurations.
-
-Use the following command to exec into the Org3CLI container:
-
-.. code:: bash
-
-  docker exec -it Org3cli bash
-
-This container has been mounted with the ``organizations`` folder, giving us
-access to the crypto material and TLS certificates for all organizations and the
-Orderer Org. We can use environment variables to operate the Org3CLI container
-as the admin of Org1, Org2, or Org3. First, we need to set the environment
-variables for the orderer TLS certificate and the channel name:
-
-.. code:: bash
-
-  export ORDERER_CA=/opt/gopath/src/github.com/hyperledger/fabric/peer/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
-  export CHANNEL_NAME=mychannel
+so that the Org3 peer resolves with the existing peers and ordering
+node of the test network. 
 
-Check to make sure the variables have been properly set:
-
-.. code:: bash
-
-  echo $ORDERER_CA && echo $CHANNEL_NAME
-
-.. note:: If for any reason you need to restart the Org3CLI container, you will also need to
-          re-export the two environment variables -- ``ORDERER_CA`` and ``CHANNEL_NAME``.
+.. note:: the `./addOrg3.sh up` command uses a `fabric-tools` CLI container to perform 
+          the channel configuration update process demonstrated below. This is to avoid the 
+          `jq` dependency requirement for first-time users. However, it is recommended to 
+          follow the process below directly on your local machine instead of using the unnecessary
+          CLI container.
 
 Fetch the Configuration
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Now we have the Org3CLI container with our two key environment variables -- ``ORDERER_CA``
-and ``CHANNEL_NAME`` exported.  Let's go fetch the most recent config block for the
-channel -- ``mychannel``.
+Let's go fetch the most recent config block for the channel -- ``channel1``.
```

Comment:
why the new channel name?

Reply:
The manual steps for the create a channel tutorial use `channel1` as the channel name. I figured it made sense to be consistent, especially if somebody wanted to manually work there way through org1 and org2 on the test-network and then add org3 manually as well.

Reply:
makes sense.